### PR TITLE
basic_login

### PR DESCRIPTION
**Pull request by weilingooi**
finish basic login

### CODE REVIEW

**Comment by weilingooi** [comment]
ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/sessions.scss]: remove empty file you have crated

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] =user.id
```
Comment (ngocvt-0484) [inline app/helpers/sessions_helper.rb]:
```suggestion
    session[:user_id] = user.id
```

File: app/helpers/sessions_helper.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  end
+  
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```
Comment (ngocvt-0484) [inline app/helpers/sessions_helper.rb]:
```suggestion
    @current_user ||= User.find_by id: session[:user_id] if session[:user_id]
```

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,12 +1,29 @@
...
+	        </ul>
+	      </li>
+	    <% else %>
+	      <li><%= link_to t("log_in"), login_path %></li>
+	    <% end %>
+      </ul> 
+    </nav>
+  </div>
+</header>
+
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb]: remove this line

File: config/routes.rb [hunk 5]
```diff
@@ -1,9 +1,13 @@
...
   root "static_pages#home"
   
-  get "static_pages/home"
-  get "static_pages/help"
+  get "home", to: "static_pages#home", as: "home"
+  get "help", to: "static_pages#help", as: "help"
   resources :users, only: %i(new create show)
   get "/signup", to: "users#new"
   post "/signup", to: "users#create"
+  get "sessions/new"
```
Comment (ngocvt-0484) [inline config/routes.rb]: remove it
it does'nt use

**Comment by weilingooi** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**